app/components/transactions/TransactionRow.js

```diff
@@ -220,29 +259,24 @@ class TransactionRow extends Component<Props, State> {
           <LeftDetails>
```

Comment:
LeftDetails and RightDetails are redundant, remove them and change DetailsSection styling if necessary

Reply:
fix